Modix.Bot/Modules/GiveawayModule.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+using Discord;
+using Discord.Commands;
+
+using Humanizer;
+
+using Modix.Services.CommandHelp;
+
+namespace Modix.Bot.Modules
+{
+    [Name("Giveaways")]
+    [Summary("Host timed or untimed giveaways.")]
+    [Group("giveaway")]
+    [Alias("giveaways")]
+    [HelpTags("giveaways")]
+    [RequireUserPermission(GuildPermission.Administrator)]
+    public class GiveawayModule : ModuleBase
+    {
+        [Command("choose")]
+        [Alias("pick")]
+        [Summary("Randomly chooses a winner for the supplied giveaway.")]
+        public async Task ChooseAsync(
+            [Summary("The giveaway message from which users will be drawn.")]
+                DiscordUserMessage message,
+            [Summary("How many winners to choose.")]
+                int count = 1)
```

Comment:
Nit; would it make more sense to make this unsigned?

Reply:
Maybe, but we'd still have to check for 0

Reply:
Definitely. I just thought making it unsigned would make it convey the purpose a bit better, not that it matters in the slightest (in reality) but yeah, as I said, nit :D

Reply:
`Random.Next(max)` takes an `int` for an argument, so just sticking with `int` simplifies the code slightly (i.e. doesn't require a cast to use as an argument). I think I'll stay with `int` unless there's a more compelling reason to switch